src/funcs.c

```diff
@@ -817,6 +836,8 @@ void filter_change()
 	get(app->STR_Filter, MUIA_String_Contents, &str);
 	DoMethod(app->LV_GenresList, MUIM_List_GetEntry, MUIV_List_GetEntry_Active, &str_gen);
 
+	printf("filter_change:%s~~~\n", str);
```

Comment:
Should probably wrap this with `#ifdef DEBUG` / `#endif` or remove it.

Reply:
Thanks for that. Removed.